start-cb.sh

```diff
@@ -386,6 +397,39 @@ bridge_osx() {
     sudo networksetup -setsearchdomains Wi-Fi service.consul node.consul
 }
 
+checkHealthOnUrl() {
```

Comment:
healthChecks are the responsibility of consul, Instead of depending on external checks, we should use http://www.consul.io/docs/agent/checks.html